draft-ietf-rats-corim.md

```diff
@@ -2626,7 +2626,7 @@ If any of the fields does not match, then the condition ECT does not match the A
 The Verifier SHALL compare each field which is present in the condition ECT `environment-map` against the corresponding field in the ACS entry `environment-map` using binary comparison.
 Before performing the binary comparison, the Verifier SHOULD convert both `environment-map` fields into a form which meets CBOR Core Deterministic Encoding Requirements {{-cbor}}.
 
-If all fields which are present in the condition ECT `environment-map` are present in the ACS entry and are binary identical, then the environments match.
+If all fields which are present in the condition ECT `environment-map` are present in the ACS entry and are binary identical (e.g., instance-id or group-id), then the environments match.
```

Comment:
The new parenthetical seems misplaced.  Should it be moved just after “`environment-map`”?